plugins/netflow.yaml

```
@@ -17,10 +17,36 @@ parameters:
       - v5
       - ipfix
     required: true
+  - name: sampling_enable
+    label: Sampling
+    description: Enable Netflow sampling
+    type: bool
+    default: false
+  - name: sampling_drop_rate
+    label: Number
```

The only thing that strikes me as odd here is this label just being `Number`. Maybe `Sampling Drop Rate` or similar?
```suggestion
    label: Sampling Drop Rate
```